### PR TITLE
refactor(datetime): replace enum with object

### PR DESCRIPTION
**Pull request by realpha**
This PR replaces the used typescript enum in the `datetime` module with a 
as `const object` as requested [here](https://github.com/denoland/deno_std/issues/3782)

As this enum is only used internally, merging this change wouldn't constitute a
breaking change.

History:
- chore(root): ignore .tool-versions
- feat(datetime): replace enum with const object

### CODE REVIEW

File: .gitignore
Comment (iuioiua) [inline .gitignore:1]: Can you please revert this change? Best we stick to the objective of the PR 🙂

**Review by iuioiua — CHANGES_REQUESTED** [review]

File: .gitignore
Comment (realpha) [reply .gitignore:1]: sure thing! Out of curiosity: Is the convention in deno_std to always use the latest `deno` cli version?

Comment (iuioiua) [reply .gitignore:1]: Yep, the latest version is always best 👍🏾

**Review by iuioiua — APPROVED** [review]
LGTM! Thank you 🙏🏾
